### PR TITLE
Remove redundant items

### PR DESCRIPTION
**Pull request by JieXiong9119**
Reference issues #271, #272, #275

### CODE REVIEW

**Review by nllong — CHANGES_REQUESTED** [review]
This is great, thanks Jie! A couple things:

1) we need to base the PR against a new `develop-v3` branch which I will create today.
2) Can you add a proposal into the proposal folder so that we know what this change is? 
3) After 2, can we also include Jan and Michael Braunch from CTS on these updates?

**Comment by JieXiong9119** [comment]
> This is great, thanks Jie! A couple things:
> 
> 1. we need to base the PR against a new `develop-v3` branch which I will create today.
> 2. Can you add a proposal into the proposal folder so that we know what this change is?
> 3. After 2, can we also include Jan and Michael Braunch from CTS on these updates?

Will do!

**Comment by JieXiong9119** [comment]
> This is great, thanks Jie! A couple things:
> 
> 1. we need to base the PR against a new `develop-v3` branch which I will create today.
> 2. Can you add a proposal into the proposal folder so that we know what this change is?
> 3. After 2, can we also include Jan and Michael Braunch from CTS on these updates?

Hey Nick, please check the added proposals for the changes.
For 3, I don't know how to add Jan and Michael to here. Should I just forward this PR in email to them?

**Comment by nllong** [comment]
Great, thanks! I added Yan as a reviewer. Can you send link to CTS? I'll slack you the email.

**Review by yan130 — APPROVED** [review]

**Review by nllong — APPROVED** [review]
Looks good. Let's wait until after meeting with CTS before merging.

File: proposals/2021/Delete Install or Upgrade Master Venting.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Delete Install or upgrade master venting
+
+## Overview
+
+This proposal is to address the issue of the redundant `Install or upgrade master venting` measure option under `ChilledWaterHotWaterAndSteamDistributionSystems` and `Install or Upgrade Master Venting` measure option under `OtherHVAC`.
+
+## Justification
+
+`Install or upgrade master venting` measure repeats in `ChilledWaterHotWaterAndSteamDistributionSystems` and `OtherHVAC`, and we should keep only one of them. This proposal suggests that we remove it under `ChilledWaterHotWaterAndSteamDistributionSystems` and keep it under `OtherHVAC`, as master venting approach could be implemented not only in chilled or hot water system.
```
Comment (JieXiong9119) [inline proposals/2021/Delete Install or Upgrade Master Venting.md:9]: @nllong could we justify whether we should keep it under `ChilledWaterHotWaterAndSteamDistributionSystems` or `OtherHVAC`? Any reference to look up?

**Review by JieXiong9119 — COMMENTED** [review]
Make decision and implement the latest proposal: delete Install or upgrade master venting

File: proposals/2021/Delete Install or Upgrade Master Venting.md (hunk 1 above)
Comment (nllong) [reply proposals/2021/Delete Install or Upgrade Master Venting.md:9]: I would check ASHRAE Standard 100. 
image

**Review by Ryoken — APPROVED** [review]

**Review by nllong — APPROVED** [review]

**Pull request merged**